core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages.component.ts

```diff
@@ -67,7 +67,24 @@ export class DotPagesComponent implements OnInit, OnDestroy {
             urlParams[entry[0]] = entry[1];
         }
 
-        this.dotRouterService.goToEditPage(urlParams);
+        this.dotPageRenderService
+            .checkPermission(urlParams)
+            .pipe(take(1))
+            .subscribe((hasPermission: boolean) => {
+                if (hasPermission) {
+                    this.dotRouterService.goToEditPage(urlParams);
+                } else {
+                    const error = new HttpErrorResponse(
+                        new HttpResponse({
+                            body: null,
+                            status: HttpCode.FORBIDDEN,
+                            headers: null,
+                            url: ''
+                        })
+                    );
+                    this.dotHttpErrorManagerService.handle(error);
+                }
```

Comment:
So the backend doesn't send the error? because if not... looks like it should.

Reply:
no, it sends a true/false on the response indicating if the user has permissions or not. @jdotcms want to do that change (send a 403 error instead of a `false`)?

Reply:
I think it is semantically ok. but even if it is 200 on true or false, what the endpoint does is to check if the user has or not the permission